### DISCUSSION TITLE
QCarouselSlide and v-bind

### DISCUSSION THREAD

**Discussion by alessioguidi** [quasarframework/quasar#11641] [General - Components / Directives / etc] [answered]
Goodmorning everyone. I state that I am a beginner with quasar and vue. I have a problem with the QCarouselSlide component.
I would like to pass the img-src property using v-bind, but it doesn't seem to work. See attached code
[quasar.txt](https://github.com/quasarframework/quasar/files/7696666/quasar.txt)

Thanks for any help

**Comment by pdanpdan** [comment]
Make a codepen. In the docs on all the examples you have an icon (middle) to open a codepen. Put your code and don't forget to save

**Reply by alessioguidi** [reply]
ok! try this
https://codepen.io/alessioguidi/pen/qBPaQWX?editors=101

**Comment by MilosPaunovic** [comment, marked as answer]
You have to set the `slide: ref('foo')` to reference the existing slide name, for `QCarousel` to know what to show.

[https://codepen.io/milospaunovic/pen/ZEXpmGW?editors=1010](https://codepen.io/milospaunovic/pen/ZEXpmGW?editors=1010)

**Comment by alessioguidi** [comment]
Ok, now it's clear
Thank you

Il giorno sab 11 dic 2021 alle ore 09:41 Milos Paunovic <
***@***.***> ha scritto:
